src/components/EventForm.js

```diff
@@ -129,14 +151,15 @@ export default class EventForm extends React.Component {
           numberOfMonths={1}
           minimumNights={0}
         />
-        {this.state.image === "" ? (
+
+        {this.state.imageUrl === "" ? (
```

Comment:
You can omit `=== ""` because the empty string is falsy. Remember to swap condition results.